src/common/InlineDateTime.tsx

```diff
@@ -16,3 +16,12 @@ export default function InlineDateTime(props: Props) {
         <span>{ formattedDateTime }</span>
     );
 }
+
+export function formatDateTime(dateTime?: string): string {
```

Comment:
Re-use this function in function `InlineDateTime` - 10 lines above.

Reply:
I rather removed the function as it is useless actually.